_posts/2016/2016-05-31-gradle-jenkins-plugin.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "How to empower your Jenkins with Gradle"
+date: "2016-05-31 10:59:11"
+categories: tech-stories
+authors: ["Renato T."]
+---
+
+Since the [last blog]( https://developer.epages.com/blog/2016/01/28/jenkins-job-dsl-plugin.html) post we did some updates in our Jenkins environments and today we are going to talk a bit about it.
+We are using a [Gradle Jenkins plugin](https://github.com/ghale/gradle-jenkins-plugin/wiki) to control Jenkins installation.
+
+We already mention about the dsl, in the [Job DSL Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Job+DSL+Plugin)
```

Comment:
We already mention about the dsl, in the [Job DSL Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Job+DSL+Plugin)
>
We already mentioned the DSL in the [Job DSL Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Job+DSL+Plugin) > Was the link supposed to be on our blog post: https://developer.epages.com/blog/2016/01/28/jenkins-job-dsl-plugin.html ?